qa/suites/rados/cephadm-smoke/distro/ubuntu_18.04_podman.yaml

```diff
@@ -0,0 +1,8 @@
+os_type: ubuntu
+os_version: "18.04"
+tasks:
+- exec:
+    all:
+    - sudo apt -y install software-properties-common
+    - sudo add-apt-repository -y ppa:projectatomic/ppa
```

Comment:
I'm using projectatomic myself here, but I don't know if we should embrace it here. Maybe something like

```yaml
# feel free to remove this test, if ppa:projectatomic is no longer maintained.
```